packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -36,9 +38,15 @@ const DOM = props => {
 		initialState = {},
 		scripts,
 		cssLinks,
+		userAgent,
 	} = props;
 
-	const htmlLang = initialState.config.requestLanguage.split('-')[0];
+	const localeCode = initialState.config.requestLanguage;
+	const htmlLang = localeCode.split('-')[0];
+
+	// Polyfill all browsers except Chrome and Firefox
+	// TODO: fine-tune this or leave is for now?
+	const doPolyfill = userAgent && userAgent.indexOf("Chrome") === -1 && userAgent.indexOf("Firefox") === -1;
```

Comment:
I recommend breaking this out into a function that can be called from here rather than adding specific functionality inline

e.g.

```js
const polyfillWhitelist = ['Chrome', 'Firefox'];
const getPolyfill = (userAgent, localeCode) = > // this is all nice and pure and unit-testable
  !polyfillWhitelist.some(userAgent.includes) &&
    polyfillServiceUrl(localeCode);

// ...
const polyfillSrc = getPolyfill(userAgent, localeCode);
```

Reply:
the `getPolyfill` function could/should actually live in the `browserPolyfill` module to reduce the amount of polyfill-related noise in `Dom.jsx`